cfpb/consumerfinance.gov
# Flatten template hierarchy and merge `content-base.html` and `layout-full.html` templates

Two pages extend the `content-base.html` template directly: https://www.consumerfinance.gov/know-before-you-owe/ and https://www.consumerfinance.gov/owning-a-home/explore-rates/

Otherwise, `layout-full.html` extends `content-base.html`, which 20 other pages use.

## Changes

- Merge `content-base.html` and `layout-full.html` templates.
- Flatten template hierarchy so that all page templates extend directly from base.html.


## How to test this PR

1. These pages use the layout full template. They should be unchanged:

Form explainer:
http://localhost:8000/owning-a-home/loan-estimate/

External link interstitial
http://localhost:8000/about-us/blog/state-regulatory-developments-on-income-based-advances/ (scroll to bottom and click an external link to get to the interstitial).

CCDB
http://localhost:8000/data-research/consumer-complaints/search/

Gov-delivery
https://www.consumerfinance.gov/govdelivery-subscribe/success/
https://www.consumerfinance.gov/govdelivery-subscribe/error/

Know Before You Owe
http://localhost:8000/know-before-you-owe/
http://localhost:8000/know-before-you-owe/compare/
http://localhost:8000/know-before-you-owe/timeline/

Rate Explorer
http://localhost:8000/owning-a-home/explore-rates/

Transcript
http://localhost:8000/transcripts/how-to-apply-for-a-federal-job-with-the-cfpb/

Mega Menu Preview
Go into Wagtail admin and go to the Mega Menu settings and hit "Preview"

Prepaid agreements
http://localhost:8000/data-research/prepaid-accounts/search-agreements/

iRegs
http://localhost:8000/rules-policy/regulations/1002/

Before You Claim
http://localhost:8000/consumer-tools/retirement/before-you-claim/

TDP activities search
http://localhost:8000/consumer-tools/educator-tools/youth-financial-education/teach/activities/

Story page
http://localhost:8000/about-us/racial-equity/
http://localhost:8000/about-us/small-business-lending/
http://localhost:8000/es/quienes-somos/prestamos-pequenos-negocios/

Recent updates
http://localhost:8000/activity-log

Spanish homepage
http://localhost:8000/es/

Homepage
http://localhost:8000/

400 and 500 error pages
Not sure how to view these locally?
